api/dsr/v1/Access.md

```diff
@@ -127,50 +129,56 @@ Content-Type: application/json
-| *response.subject*                     | no        | The complete [Data Subject](README.md#Subject) to replace the Data Subject details or an array of [JSON Patch](https://jsonpatch.com/) specification of the Data Subject details to mutate. |
-| *response.identities*                  | no        | Array of [Identities](README.md#Identity) to add to the request                                                                                                                             |
-| *response.messages*                    | no        | Array of [Messages](README.md#Message) to add to the conversation                                                                                                                           |
+| name                          | required? | description                                                                                                                                                                                 |
+|-------------------------------|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| *status*                      | yes       | The [status](Status.md#Status code) of the Data Subject Request                                                                                                                             |
+| *reason*                      | no        | The [reason](Status.md#Reason) for the status of the Data Subject Request. Default is `other`.                                                                                              |
+| *expectedCompletionTimestamp* | no        | The UNIX timestamp at which the Data Subject Request is expected to be completed                                                                                                            |
+| *requestID*                   | no        | The request ID known to the destination system                                                                                                                                              |
+| *results*                     | no        | Array of [Documents](README.md#Document) that can be used to download the contents requested                                                                                                |
+| *documents*                   | no        | Array of [Documents](README.md#Document) that can be used to download the contents requested                                                                                                |
+| *claims*                      | no        | Map containing additional claims that are to be added                                                                                                                                       |
```

Comment:
Need to support updating / removing claims?

Reply:
Update by returning same key. Remove by using `null` as value

Reply:
Or you mean the text?

Reply:
That makes sense. Is there a reason why data subject should behave differently w json patch?

```| *subject* | no | The complete [Data Subject](README.md#Subject) to replace the Data Subject details or an array of [JSON Patch](https://jsonpatch.com/) specification of the Data Subject details to mutate. |```